src/open_inwoner/accounts/views/registration.py

```diff
@@ -146,6 +146,8 @@ def get_object(self, queryset=None):
 
     def get_success_url(self):
         messages.add_message(self.request, messages.SUCCESS, "Registratie is voltooid")
+        if "next" in self.request.GET:
+            return self.request.GET["next"]
```

Comment:
Here we need a `is_safe_url` (or in Django 4.0 `url_has_allowed_host_and_scheme`) to protect against unsafe redirects